lib/main.coffee

```diff
@@ -1,6 +1,5 @@
-{CompositeDisposable} = require 'atom'
 ReleaseNotesView = null
-ReleaseNoteStatusBar = require  './release-notes-status-bar'
+ReleaseNoteStatusBar = null
```

Comment:
The `ReleaseNoteStatusBar` variable doesn't need to be in this outer scope anymore. I think you should remove this line and change the `?=` to an `=` on line where it is assigned.